docker/base/entry.sh

```diff
@@ -9,4 +9,4 @@ fi
 
 echo $@
 # We can't use start:headless here bcs that task hardcodes the .js file, we want it to be extensible
-node --experimental-vm-modules --experimental-wasm-modules --es-module-specifier-resolution=node $@
\ No newline at end of file
+node --experimental-vm-modules --experimental-wasm-modules --experimental-wasm-threads --es-module-specifier-resolution=node $@
```

Comment:
iirc you removed the experimental flag in 3 places, make sure to add back in all of them